docs/designers-developers/developers/themes/theme-support.md

```diff
@@ -216,6 +216,8 @@ As an example for the regular font size, a theme may provide the following class
 }
 ```
 
+**Note:** The slugs `default` and `custom` are reserved and cannot be used by themes.
```

Comment:
I guess we should also document this in https://github.com/WordPress/gutenberg//blob/6d4382efa125d385e57bfc835057bd0c44a51347/docs/designers-developers/developers/themes/theme-support.md#L177.

Reply:
Is not this PR already documenting it on that section or am I missing something?

Reply:
I'm sorry I posted the wrong link. I wanted to refer the readme of the component `packages/components/src/font-size-picker/README.md`. The component also receives slugs and these slugs should also not be the reserved ones.

Reply:
Gotcha. Added there too 15f0fd9